src/thirdparty/CMakeLists.txt

```diff
@@ -126,11 +126,13 @@ if (AXOM_ENABLE_SPARSEHASH)
                 $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/thirdparty>)
 
     # Disable warning introduced in gcc@8.1 related to how sparsehash casts memory
-    if(C_COMPILER_FAMILY_IS_GNU)
-        if(CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL "8.1")
-            blt_add_target_compile_flags(TO sparsehash FLAGS 
-              $<$<NOT:$<COMPILE_LANGUAGE:Fortran>>:-Wno-class-memaccess>)
-        endif()
+    if(C_COMPILER_FAMILY_IS_GNU AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL "8.1")
```

Comment:
Thanks @corbett5.  This should fix the problem.  It's pretty unwieldy though.

Can anyone think of a better way to do this? 
Tag: @white238 @davidbeckingsale

Reply:
I agree, especially since this logic gets sprinkled elsewhere in #4.
https://github.com/LLNL/axom/pull/4#discussion_r278293586